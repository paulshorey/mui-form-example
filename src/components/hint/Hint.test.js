import React from 'react';
import renderer from 'react-test-renderer';

// shallow + redux
import { shallow, connected, shallowConnected } from 'index.testing';
import { Hint } from './Hint';

it('renders correctly', () => {
	const tree = renderer
		.create(
			shallowConnected(
				<Hint title="the title" description={<p>This can be a react component!</p>} />
			)
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
