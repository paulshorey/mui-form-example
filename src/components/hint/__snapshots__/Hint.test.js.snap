// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Hint sc-bdVaJa diGFYN"
  onClick={[Function]}
>
  <sup
    className="fontIcon icon-pk_question"
  />
</div>
`;
